scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -466,19 +466,33 @@ END;
 $$ LANGUAGE 'plpgsql';
 
 
+
+DO $$
+BEGIN
+  IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = '_cdb_has_usable_primary_id_record') THEN
+    CREATE TYPE _cdb_has_usable_primary_id_record
```

Comment:
Agree :-) I had some issues late yesterday with the returns and one of my attempts was to force this, but it's not needed anymore.